agents/components/SelectAgentForOrder.js

```diff
@@ -53,13 +56,18 @@ function SelectAgentForOrder (props) {
       }, 'create new')
     ])
   ])
+})
+
+const SelectAgentForOrder = (props) => {
+  const nextProps = merge(props, {
+    form: `select${props.agentType}`,
```

Comment:
is this for adherence to a common convention, or something else i'm missing? @ahdinosaur

Reply:
`selectsupplier` is hard to read as "select supplier". there's many possible conventions we could choose from: we could use camelCase, but to avoid capitalization pain and since it's common in DOM values in this case i've been using hypen-case

Reply:
👍